appveyor.yml

```diff
@@ -3,6 +3,9 @@
 
   branches:
     only:
+      - release
```

Comment:
What does release represent here ?

Reply:
Pulled from upstream, I will remove it.  I am assuming they are using release branching which we do not use.